### ISSUE TITLE
bb shows black screen and uses 100% CPU when terminal window size is small at startup

### ISSUE THREAD

**Issue by ghuls** [epilys/bb#7]
bb shows black screen and uses 100% CPU (and can't be closed with `q`) when terminal window size is small at startup.

Starting bb in a terminal with a bigger terminal size and then resizing the terminal size to the small size works fine.

**Comment by epilys** [comment]
Are you using the latest version? (0.3.2) Also how small should the terminal size be to trigger this bug? I haven't the same behavior in my terminal, which one are you using?

**Comment by ghuls** [comment]
I just tried the last git version. The 100% CPU usage is gone, but the no text is still shown when the terminal is too small.

When I make the terminal a bit bigger, there is another artifact. The CPU% column displays NaNs. This server has 36 CPUs. Once I make the terminal large enough (all CPU columns start showing up) the CPU% column displays the correct values. 

![image](https://user-images.githubusercontent.com/1299177/68114912-7aac4c00-fef7-11e9-8921-b735b7b6668b.png)

![image](https://user-images.githubusercontent.com/1299177/68115107-e8f10e80-fef7-11e9-8a19-dafb521c4a32.png)

**Comment by epilys** [comment]
Is the size threshold at about 80x20? Smaller resolutions than that shouldn't work (they're too small anyway) but I suspect you're talking about larger ones.

I'll look into the CPU time value problem, weird.

**Comment by ghuls** [comment]
Now I was able to reproduce the 100% CPU usage by `bb`.

Here are the column and line values from which `bb` has problems. The values seem to change (maybe depending on the load on the server.)
```
# Still shows some stuff on screen (no CPU columns)
$ echo $COLUMNS $LINES
99 26

# Completely empty screen and bb is frozen (need to use kill -9, 'q' and Ctrl+C does not work).
$ echo $COLUMNS $LINES
98 26


# Completely empty screen and bb is frozen (need to use kill -9, 'q' and Ctrl+C does not work).
$ echo $COLUMNS $LINES
104 26

# Shows some stuff on screen (with CPU columns)
$ echo $COLUMNS $LINES
105 26
```

**Comment by epilys** [comment]
Hey @ghuls, I'm so sorry for not replying earlier. I usually put reminders for this stuff because I forget easily but it seems `bb` was left out of this routine.

I was unable to replicate this, I used the same and then some other various values in my own terminal (`st`). It'd be helpful if you gave me a backtrace, read the instructions I wrote on this other issue https://github.com/epilys/bb/issues/16#issuecomment-581197609
